[1.10] Crash on server startup

SimplyConveyors: 1.4.0
Forge: 12.18.3.2202

```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Simply Conveyors (simplyconveyors)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/color/IBlockColor
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at net.minecraftforge.fml.common.FMLModContainer.constructMod(FMLModContainer.java:505)
	at sun.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145)
	at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:549)
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:97)
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:328)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:121)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.renderer.color.IBlockColor
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 29 more
Caused by: java.lang.NullPointerException
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
	... 31 more
```

Are you sure you aren't referring to both of these mods?

I can't find a deobf version of mcjtylib or your furniture mod, by the way.

Maybe when you get home, check your `libs/` dir or see what you've installed to your local maven repo?

---

```
C:\Users\-gdude2002\IdeaProjects\Simply-Conveyors\build\sources\main\java\com\momnop\simplyconveyors\SimplyConveyorsCreativeTab.java:11: error: package com.momnop.aquastuff.blocks does not exist
import com.momnop.aquastuff.blocks.AquaStuffBlocks;
                                  ^
C:\Users\-gdude2002\IdeaProjects\Simply-Conveyors\build\sources\main\java\com\momnop\simplyconveyors\blocks\conveyors\special\BlockMovingBackwardsDropperPath.java:3: error: package mcjty.lib.tools does not exist
import mcjty.lib.tools.ItemStackTools;
                      ^
C:\Users\-gdude2002\IdeaProjects\Simply-Conveyors\build\sources\main\java\com\momnop\simplyconveyors\blocks\conveyors\special\BlockMovingDropperPath.java:3: error: package mcjty.lib.tools does not exist
import mcjty.lib.tools.ItemStackTools;
                      ^
C:\Users\-gdude2002\IdeaProjects\Simply-Conveyors\build\sources\main\java\com\momnop\simplyconveyors\blocks\conveyors\special\BlockMovingTransporterPath.java:3: error: package mcjty.lib.tools does not exist
import mcjty.lib.tools.ItemStackTools;
                      ^
C:\Users\-gdude2002\IdeaProjects\Simply-Conveyors\build\sources\main\java\com\momnop\simplyconveyors\helpers\ConveyorHelper.java:5: error: package mcjty.lib.tools does not exist
import mcjty.lib.tools.ItemStackTools;
```

--- a/src/main/java/com/momnop/simplyconveyors/SimplyConveyors.java
+++ b/src/main/java/com/momnop/simplyconveyors/SimplyConveyors.java
@@ -1,17 +1,5 @@
 package com.momnop.simplyconveyors;
 
-import net.minecraft.block.Block;
-import net.minecraft.block.state.IBlockState;
-import net.minecraft.client.Minecraft;
-import net.minecraft.client.renderer.color.IBlockColor;
-import net.minecraft.client.renderer.color.IItemColor;
-import net.minecraft.init.Blocks;
-import net.minecraft.item.EnumDyeColor;
-import net.minecraft.item.ItemStack;
-import net.minecraft.util.math.BlockPos;
-import net.minecraft.world.ColorizerFoliage;
-import net.minecraft.world.IBlockAccess;
-import net.minecraft.world.biome.BiomeColorHelper;
 import net.minecraftforge.common.ForgeChunkManager;
 import net.minecraftforge.common.MinecraftForge;
 import net.minecraftforge.fml.common.Mod;
@@ -28,14 +16,12 @@ import net.minecraftforge.fml.common.network.NetworkRegistry;
 import net.minecraftforge.fml.common.registry.GameRegistry;
 import net.minecraftforge.fml.relauncher.Side;
 
-import com.momnop.furniture.blocks.BlockSofa;
 import com.momnop.simplyconveyors.blocks.SimplyConveyorsBlocks;
 import com.momnop.simplyconveyors.blocks.bus.tiles.TileEntityBusStop;
 import com.momnop.simplyconveyors.blocks.conveyors.tiles.TileEntityDetectorBackwardsPath;
 import com.momnop.simplyconveyors.blocks.conveyors.tiles.TileEntityDetectorPath;
 import com.momnop.simplyconveyors.blocks.conveyors.tiles.TileEntityGrabberPath;
 import com.momnop.simplyconveyors.blocks.conveyors.tiles.TileEntityTransporterPath;
-import com.momnop.simplyconveyors.blocks.roads.BlockConnectingColored;
 import com.momnop.simplyconveyors.client.render.blocks.BlockRenderRegister;
 import com.momnop.simplyconveyors.client.render.guis.SimplyConveyorsGuiHandler;
 import com.momnop.simplyconveyors.config.ConfigHandler;
@@ -197,34 +183,11 @@ public class SimplyConveyors
 		
 		if(event.getSide() == Side.CLIENT)
 		{
-			registerColored(SimplyConveyorsBlocks.blockBrokenRoad);
-			registerColored(SimplyConveyorsBlocks.blockFullRoad);
+			proxy.registerColored(SimplyConveyorsBlocks.blockBrokenRoad);
+			proxy.registerColored(SimplyConveyorsBlocks.blockFullRoad);
 			
 			BlockRenderRegister.registerBlockRenderer();
 		}
-	}
-	
-	public static void registerColored(Block block) {
-		Minecraft.getMinecraft().getBlockColors().registerBlockColorHandler(new IBlockColor() {
-            @Override
-            public int colorMultiplier(IBlockState iBlockState, IBlockAccess iBlockAccess, BlockPos blockPos, int i) {
-                if(iBlockState != null){
-                	return EnumDyeColor.byMetadata(iBlockState.getValue(BlockConnectingColored.COLOR).getMetadata()).getMapColor().colorValue;
-                }
-                return -1;
-            }
-        }, block);
-		
-		Minecraft.getMinecraft().getItemColors().registerItemColorHandler(new IItemColor() {
-            @Override
-            public int getColorFromItemstack(ItemStack stack, int tintIndex)
-            {
-            	if (stack != ItemStack.EMPTY) {
-            		return EnumDyeColor.byMetadata(stack.getMetadata()).getMapColor().colorValue;
-            	}
-            	return -1;
-            }
-        }, block);
 	}
 
 	@EventHandler

--- a/src/main/java/com/momnop/simplyconveyors/SimplyConveyorsCreativeTab.java
+++ b/src/main/java/com/momnop/simplyconveyors/SimplyConveyorsCreativeTab.java
@@ -8,7 +8,6 @@ import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.NonNullList;
 
-import com.momnop.aquastuff.blocks.AquaStuffBlocks;
 import com.momnop.simplyconveyors.blocks.SimplyConveyorsBlocks;
 import com.momnop.simplyconveyors.info.ModInfo;
 import com.momnop.simplyconveyors.items.SimplyConveyorsItems;

--- a/src/main/java/com/momnop/simplyconveyors/proxies/ClientProxy.java
+++ b/src/main/java/com/momnop/simplyconveyors/proxies/ClientProxy.java
@@ -1,10 +1,19 @@
 package com.momnop.simplyconveyors.proxies;
 
+import com.momnop.simplyconveyors.blocks.roads.BlockConnectingColored;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
 import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.color.IBlockColor;
+import net.minecraft.client.renderer.color.IItemColor;
 import net.minecraft.client.renderer.entity.Render;
 import net.minecraft.client.renderer.entity.RenderManager;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.EnumDyeColor;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
 import net.minecraftforge.fml.client.registry.ClientRegistry;
 import net.minecraftforge.fml.client.registry.IRenderFactory;
 import net.minecraftforge.fml.client.registry.RenderingRegistry;
@@ -73,5 +82,28 @@ public class ClientProxy extends CommonProxy
 
 			return renderer;
 		}
+	}
+
+	public void registerColored(Block block) {
+		Minecraft.getMinecraft().getBlockColors().registerBlockColorHandler(new IBlockColor() {
+			@Override
+			public int colorMultiplier(IBlockState iBlockState, IBlockAccess iBlockAccess, BlockPos blockPos, int i) {
+				if(iBlockState != null){
+					return EnumDyeColor.byMetadata(iBlockState.getValue(BlockConnectingColored.COLOR).getMetadata()).getMapColor().colorValue;
+				}
+				return -1;
+			}
+		}, block);
+
+		Minecraft.getMinecraft().getItemColors().registerItemColorHandler(new IItemColor() {
+			@Override
+			public int getColorFromItemstack(ItemStack stack, int tintIndex)
+			{
+				if (stack != ItemStack.EMPTY) {
+					return EnumDyeColor.byMetadata(stack.getMetadata()).getMapColor().colorValue;
+				}
+				return -1;
+			}
+		}, block);
 	}
 }

--- a/src/main/java/com/momnop/simplyconveyors/proxies/CommonProxy.java
+++ b/src/main/java/com/momnop/simplyconveyors/proxies/CommonProxy.java
@@ -1,5 +1,6 @@
 package com.momnop.simplyconveyors.proxies;
 
+import net.minecraft.block.Block;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
 import net.minecraftforge.fml.common.registry.GameRegistry;
@@ -36,6 +37,10 @@ public class CommonProxy
 
 	public void initKeybinds()
 	{
+
+	}
+
+	public void registerColored(Block block) {
 
 	}
 

